src/spec/__tests__/log.test.ts

```diff
@@ -0,0 +1,108 @@
+import { ILogMsg, Logger } from '../../common/logger';
```

Comment:
Should remove `Logger` as it is no longer exported 
```suggestion
import { ILogMsg } from '../../common/logger';
```